config/apparmor/abstractions/container-base.in

```diff
@@ -86,6 +86,7 @@
   mount options=(rw, nosuid, nodev, noexec, remount) -> /sys/,
   deny /sys/firmware/efi/efivars/** rwklx,
   deny /sys/kernel/security/** rwklx,
+  mount options=(move) /sys/fs/cgroup/cgmanager/ -> /sys/fs/cgroup/cgmanager.lower/,
```

Comment:
Since cgmanager is dead why would we add this to the container profile now?

Reply:
Just because this line currently in `config/apparmor/abstractions/container-base`. As I've written:
```
Commit ("cgmanager: container-base apparmor abstraction: allow mount move")
https://github.com/lxc/lxc/commit/c08a0b7c4e459f32a939391bc5c9667eb7c7ab5c
directly modifies config/apparmor/abstractions/container-base file,
but not the template file container-base.in.

Let's make things consistent.
```

Let's remove that rule everywhere then :)